roles/installer/templates/deployment.yaml.j2

```diff
@@ -222,6 +222,10 @@ spec:
               mountPath: "/var/run/awx-rsyslog"
             - name: rsyslog-dir
               mountPath: "/var/lib/awx/rsyslog"
+            - name: "{{ meta.name }}-receptor-config"
+              mountPath: "/etc/receptor/receptor.conf"
+              subPath: receptor.conf
+              readOnly: true
             - name: receptor-socket
               mountPath: "/var/run/receptor"
```

Comment:
That socket gets created by the `ee` container (https://github.com/ansible/awx-operator/blob/devel/roles/installer/templates/deployment.yaml.j2#L265) however, it seems the same socket gets mounted on the `task` container which allows the `task` container to issue commands. 

@rooftopcellist  If you still have the environment, are you able to connect on the `task` container and then do:

```bash
nc -U /var/run/receptor/receptor.sock

OR
RECEPTORCTL_SOCKET=/var/run/receptor/receptor.sock receptorctl status
```

Reply:
@tchellomello Yep, the .sock file is created where we expect it to be:

```
bash-4.4$ ls -la /var/run/receptor/receptor.sock
srw-rw----. 1 awx 1000670000 0 Sep  8 12:48 /var/run/receptor/receptor.sock

```

Receptorctl status:

```
bash-4.4$ source /var/lib/awx/venv/awx/bin/activate
(awx) bash-4.4$ RECEPTORCTL_SOCKET=/var/run/receptor/receptor.sock receptorctl status
Node ID: awx-59fd4465c6-zp9qn
Version: -0.git20210902.
System CPU Count: 4
System Memory MiB: 15013

Node         Service   Type       Last Seen             Tags
awx-59fd4465c6-zpbash-4.4$ ls -la /var/run/receptor/receptor.sock
srw-rw----. 1 awx 1000670000 0 Sep  8 12:48 /var/run/receptor/receptor.sock
9qn control   Stream     2021-09-08 12:54:12   {'type': 'Control Service'}

Node         Work Types
awx-59fd4465c6-zp9qn local, kubernetes-runtime-auth, kubernetes-incluster-auth
```

I wasn't able to do the `nc` command to connect because we don't have nc in the container.